sandbox/src/routes/document_reference.py

```diff
@@ -1,20 +1,38 @@
 from urllib.parse import parse_qs
-
 from fastapi import APIRouter, Request
-
+from starlette.responses import Response
```

Comment:
We're using `fastapi`. This dependency doesn't look right. see for example https://github.com/NHSDigital/booking-and-referral-fhir-api/blob/master/sandbox/src/routes/appointment.py#L4